sdk/python/kfp/components/_python_op.py

```diff
@@ -535,6 +538,15 @@ def _func_to_component_spec_v2(
 
         arguments.extend(arguments_for_input)
 
+    # Add output placeholders for return values from func.
+    func_outputs = [
+        output for output in component_outputs
+        if output._passing_style is None
+    ]
+    for output in func_outputs:
+        flag = "--" + output.name.replace("_", "-")
```

Comment:
1) How do we prevent name clashes? For example, there can be input called `model` and output called `model`?
2) Do we want to add the `-path` suffix to the name to make the names more readable?

```
--loss /path/to/loss/file
```
vs.
```
--loss-path /path/to/loss/file
```

Adding `-output-path` suffix can solve both of those issues:
```
--loss-output-path /path/to/loss/file
```

WDYT?

Reply:
Done.